Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,30 @@
+using UnityEngine;
+
+public sealed class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private CharacterController characterController;
+    [SerializeField] private float movingSpeed;
+
+
+    private Vector2 _moveDirection;
+    private float _horizontal;
+    private float _vertical;
+
+    private void Update()
+    {
+        SetInput();
+        SetMoveDirection();
+    }
+
+    private void SetInput()
+    {
+        _horizontal = Input.GetAxisRaw("Horizontal");
+        _vertical = Input.GetAxisRaw("Vertical");

```

Comment:
Alternatively, you could swap the _horizontal and _vertical variables with a vector2, something called "_movementInput" and just set the x and y of that instead!